cardinal/server/handler/cql.go

```diff
@@ -26,13 +26,14 @@ type CQLQueryResponse struct {
 
 // PostCQL godoc
 //
-//	@Summary		Query the ecs with CQL (cardinal query language)
-//	@Description	Query the ecs with CQL (cardinal query language)
-//	@Accept			application/json
-//	@Produce		application/json
-//	@Param			cql	body		CQLQueryRequest	true	"cql (cardinal query language)"
-//	@Success		200	{object}	CQLQueryResponse
-//	@Router			/cql [post]
+//	@Summary      Query game entities with CQL (cardinal query language)
+//	@Description  Runs the provided CQL query on Cardinal and return the results
```

Comment:
The summary and description say the same thing: I'd much rather it be uniform. The description you wrote should be used for the summary here.